developer-docs-site/docs/tools/aptos-cli-tool/aptos-db-restore.md

```diff
@@ -0,0 +1,66 @@
+
+- Quickly bootstrap a database to start a new or failed node
+- Efficiently recover data from any specific period
+
+Our DB restore tool enables you to use existing public backup files to restore the database on your local machine. These public backup files, which have cryptographic proof, are stored on both AWS and Google Cloud for public download. You can use these backup files, along with our restore tool, to restore your database to any historical range or to the latest version.
+
+## **Restore DB using the Public Backup Files**
+
+Our CLI supports restoring a database using backup files. It reads from the backup files and recreates the Aptos DB. We support two kinds of restore: (1) recreating a DB with minimal transaction history at a user-specified transaction version (or the latest version the backup has), and (2) restoring the database over a specific period. In addition to (1), this option also ensures that the recreated DB carries the ledger history of the user-designated version range.
+
+**Bootstrap DB**
+
+The command restores the database from the closest snapshot to the target version. This command can quickly restore a database to a target version, but it does not restore all the transaction history from the past.
+
+Here is an example command (note: depending on whether you use AWS or Google Cloud, you may need to follow the instructions to install [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) or [gsutil](https://cloud.google.com/storage/docs/gsutil_install) as prerequisites):
+
+```bash
+# This requires follow prerequsites
+# 1. aws cli or google gsutil installed
+# 2. aptos cli 1.0.14	
+aptos node bootstrap-db \ 
+    --target-version 500000000 \
+    --command-adapter-config /path/to/s3-public.yaml \
+    --target-db-dir /path/to/local/db
+```
+
+The s3-public.yaml ([link](https://github.com/aptos-labs/aptos-networks/blob/main/testnet/backups/s3-public.yaml)) used in the command specifies the location of the public backup files mentioned above, as well as the commands used by our backup and restore tool to interact with S3. Additionally, you can use the public Google backup files as shown here [link](https://github.com/aptos-labs/aptos-networks/blob/main/testnet/backups/gcs.yaml).
+
+**Restoring a Database over a Specific Time Period**
+
+We also support restoring the database to a previous period in the past. The command will restore all transaction history (events, write sets, key-value pairs, etc.) within the specified period, along with the state Merkle tree at the target version.
+
+To use this command, you need to specify the `ledger-history-start-version` and `target-version` to indicate the period you are interested in.
+
+```bash
+# This requires aws cli installed and aptos cli 1.0.14	
+aptos node bootstrap-db \ 
+    --ledger-history-start-version 150000000 \
```

Comment:
Also, I assume this will sync from `150000000` to `155000000`? Can we mention that explicitly?